src/zenml/zen_stores/schemas/pipeline_run_schemas.py

```diff
@@ -252,8 +250,6 @@ def to_model(
                 workspace=self.workspace.to_model(),
                 user=self.user.to_model(True) if self.user else None,
                 orchestrator_run_id=self.orchestrator_run_id,
-                enable_cache=self.enable_cache,
-                enable_artifact_metadata=self.enable_artifact_metadata,
```

Comment:
Same thing here: these values are not in the `PipelineConfiguration` object in older pipeline run DB entries. Don't you need to copy them there to ensure they are not lost ?